guardian/atoms-rendering
# Share quiz buttons

## What does this change?
Add share buttons for Knowledge Quiz

## How to test
Go through Personality Quiz (on storybook), on submit you should see the results. You should also see the differnt types of submit buttons present

## Images
<img width="463" alt="Screenshot 2021-01-18 at 13 28 05" src="https://user-images.githubusercontent.com/8831403/104921346-15743780-5991-11eb-9d17-0e2c4cb63a7f.png">
